scripts/heat/instances.yaml

```diff
@@ -39,6 +39,16 @@ resources:
       flavor: { get_param: flavor }
       networks:
         - port: { get_resource: server_port }
+      user_data_format: RAW
+      user_data:
+        #!/bin/sh
+        cat >> /usr/bin/ping_forever << EOF
+        #!/bin/sh
+        trap "" 1 2 3 15
+        ping 8.8.8.8 | tee /var/log/ping_outside.out
```

Comment:
But pinging VM from crobwar node would not be defined in the heat template. So far I had no idea how to do it, maybe we'll need to start extra script directly from mkcloud?